opennms-doc/guide-admin/src/asciidoc/text/telemetryd/protocols/openconfig.adoc

```diff
@@ -89,8 +89,8 @@ Use multiple packages to customize the parameter's subsets of nodes.
 
 The `filter` element is optional; if missing, all services with the given name will be considered.
 
-Parameters can also grouped into multiple groups when different paths need to be streamed at different frequencies using
-`group` element in parameter. Global parameters like port and connection specific parameters doesn't need any group.
+Use the group element to group parameters into multiple groups when you need to stream different paths at different frequencies using
+`group` element in parameter. Global parameters like port, and connection-specific parameters, don't need any group.
```

Comment:
@cgorantla ... Sorry my edit was confusing, but it went over two lines. This section should read like this:

Use the `group` element to group parameters into multiple groups when you need to stream different paths at different frequencies. Global parameters like port, and connection-specific parameters, don't need any group.

Reply:
my bad, didn't read the sentence properly.